e2e/testsuite/testsuite.go

```diff
@@ -88,16 +87,24 @@ func (s *E2ETestSuite) GetRelayerUsers(ctx context.Context, chainOpts ...ChainOp
 // using the given channel options. The relayer returned by this function has not yet started. It should be started
 // with E2ETestSuite.StartRelayer if needed.
 // This should be called at the start of every test, unless fine grained control is required.
-func (s *E2ETestSuite) SetupChainsRelayerAndChannel(ctx context.Context, channelOpts ...func(*ibc.CreateChannelOptions)) (ibc.Relayer, ibc.ChannelOutput) {
-	chainA, chainB := s.GetChains()
+func (s *E2ETestSuite) SetupChainsRelayerAndChannel(ctx context.Context, channelOpts func(*ibc.CreateChannelOptions), chainSpecOpts ...ChainOptionConfiguration) (ibc.Relayer, ibc.ChannelOutput) {
+	chainA, chainB := s.GetChains(chainSpecOpts...)
```

Comment:
My main concern there is that we don't really need an additional function (yet!), since we either configure everything, or we do just the chains like in the new test that's being added.

I'm hoping the diff wasn't too bad 😅

Reply:
It's alright. Makes sense!